EEex/copy/EEex_Sprite.lua

```diff
@@ -553,6 +553,64 @@ function EEex_Sprite_GetCastTimerPercentage(sprite)
 end
 CGameSprite.getCastTimerPercentage = EEex_Sprite_GetCastTimerPercentage
 
+EEex_SpellIDSType = {[1] = "SPPR", [2] = "SPWI", [3] = "SPIN", [4] = "SPCL"}
```

Comment:
**Style:** Make local to `EEex_Sprite_GetSpellRES` or rename to `EEex_Sprite_Private_SpellIDSType`